src/url.js

```diff
@@ -65,16 +65,23 @@ export function parseUrl(url) {
  * Appends the string just before the fragment part of the URL.
```

Comment:
Not sure if we want to change the comment since it supports adding to the front of the url now